### ISSUE TITLE
Build fails for one commit; subsequent commits are not built

### ISSUE THREAD

**Issue by fusion809** [tlienart/Franklin.jl#540]
Hi,

I made a little stuff up with my repo's deploy.yml file by including in Pkg.add my repo's package and the build consequently failed for that commit. Then I removed that stuff up from the deploy.yml and pushed that commit and GitHub Actions hasn't built that commit (which was over 10 hours ago). I pushed an additional commit (also 10 hours ago) and GitHub Actions refused to build that one too. Here's my repo: https://github.com/fusion809/fusion809.github.io. Any ideas how I can get GitHub Actions to start deploying my website again?

Thanks for your time.

**Comment by tlienart** [comment]
can you go to https://github.com/fusion809/fusion809.github.io/actions and there should be a button saying something like "re-launch job" (or something along those lines), can you then show the output here?

**Comment by fusion809** [comment]
Nothing like that there, I searched the entire page for "job" and "launch" and no matches. I can't see anything like that anywhere on the page either.

**Comment by fusion809** [comment]
I know that CompatHelper and TagBot only seem to run once a day ~00:00-01:00 UTC, so it's possible around that time this build too will be run automatically. I hope so, as otherwise I'm a little stumped for what to do.

**Comment by fusion809** [comment]
Hip-hip hooray, I just pushed a new commit and it's building!

**Comment by fusion809** [comment]
And the build succeeded. Thanks for your help.

**Comment by tlienart** [comment]
I’m glad it worked,

There’s a “rerun jobs” button here’s a screenshot by someone else than me on another repo 

![image](https://user-images.githubusercontent.com/10897531/86083661-30d23080-ba9b-11ea-8857-3eda81555107.jpeg)

But I’m not calling you out, it’s possible that it’s not always there, there are cases where I also couldn’t find it, I don’t know GitHub enough to know what’s going on there

**Comment by fusion809** [comment]
Didn't appear for me, I just looked at it and this is what I see:

![Screenshot_20200630_143203](https://user-images.githubusercontent.com/4717341/86083880-bfb46c80-bade-11ea-8acc-8b7b44cd936c.png)

I went into the erroring commit (in case that was what you meant) and this is what I see:

![Screenshot_20200630_143417](https://user-images.githubusercontent.com/4717341/86083966-fab6a000-bade-11ea-8fe3-61b5c1a4bfe5.png)

No re-run jobs options that I can see. I wonder if this option only appears for people that have pro (i.e. paid) GitHub accounts.
